packages/wallets/src/features/cfd/screens/MT5ResetPasswordModal/MT5ResetPasswordModal.tsx

```diff
@@ -120,13 +120,17 @@ const MT5ResetPasswordModal: React.FC<TProps> = ({
                         </div>
                         <div className='wallets-mt5-reset__footer'>
                             <WalletButtonGroup isFlex isFullWidth={isMobile}>
-                                <WalletButton onClick={sendEmailVerification} size='lg' variant='outlined'>
+                                <WalletButton
+                                    onClick={sendEmailVerification}
+                                    size={isMobile ? 'lg' : 'md'}
```

Comment:
cant wait to migrate away from javascript-driven css... adds tons of unnecesary complextiy to code